README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Write the guide to setup locally

Reply:
added the guide with all the necessary steps to set it up locally, will soon be adding contributing guidelines as well.